src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription(){

```

Comment:
Perhaps we could standardise the spaces between the function definitions and the brackets